src/js/index.js

```diff
@@ -27,38 +27,46 @@ export const hpe = deepFreeze({
       'neutral-4': undefined,
       brand: 'green!',
       background: {
-        dark: '#1A1F2B',
+        dark: '#263040',
         light: '#FFFFFF',
       },
       'background-back': {
-        dark: '#1A1F2B',
+        dark: '#263040',
         light: '#EFEFEF',
       },
       'background-front': {
-        dark: '#354050',
+        dark: '#404B5C',
         light: '#FFFFFF',
       },
       'background-contrast': {
-        dark: '#FFFFFF1F',
-        light: '#CCCCCC99',
+        dark: '#FFFFFF14',
+        light: '#0000000A',
       },
       icon: 'text',
       text: {
-        dark: '#DDDDDD',
+        dark: '#C0CADC',
         light: '#444444',
       },
       'text-strong': {
         dark: '#FFFFFF',
         light: '#000000',
       },
       'text-weak': {
-        dark: '#AAAAAA',
-        light: '#666666',
+        dark: '#606B7D',
+        light: '#BBBBBB',
       },
       border: {
-        dark: '#DDDDDD',
+        dark: '#7887A1',
+        light: '#999999',
+      },
+      'border-strong': {
+        dark: '#AFBCD2',
         light: '#666666',
       },
+      'border-weak': {
+        dark: '#606B7D',
+        light: '#BBBBBB',
+      },
```

Comment:
I commented it earlier and didn't see a direct reply so reposting
>Just to confirm, were the new colors tested on the application side prior to the PR submission? Just to be sure we are not introducing new contrast issues.

Reply:
Bringing over a discussion from Slack:
Using text color of `text` on the dark colors doesn’t meet contrast requirements, but changing the text color to `text-strong` passes everything. Feels like the site will need some guidance of when to use text vs text-strong to avoid the first error

`text` on dark color palette:
![Screen Shot 2020-04-23 at 10 04 59 AM](https://user-images.githubusercontent.com/12522275/80166523-0479cf80-8593-11ea-9828-51e1efcbcea8.png)

`text-strong` on dark color palette:
![Screen Shot 2020-04-23 at 10 13 01 AM](https://user-images.githubusercontent.com/12522275/80166539-0d6aa100-8593-11ea-98d0-226ee527db37.png)

Reply:
As long as we provide appropriate guidance of how to use text colors, then we are fine from a color contrast perspective.